### DISCUSSION TITLE
Potion Effect Duration

### DISCUSSION THREAD

**Discussion by KyllianGamer** [Minestom/Minestom#1762] [Help] [answered]
Whenever I set the duration of a potion effect, the actual duration is completely different. For example:
`player.addEffect(new Potion(PotionEffect.STRENGTH, (byte) 0, 200, Potion.ICON_FLAG));` has a duration value of 200 but somehow results in like 9-ish seconds. I'm also not doing anything weird with timings in the code, it's basically just a kill event and then give potion effect.

**Comment by ZakShearman** [comment, marked as answer]
The duration is in ticks, so that's 200 ticks (10 seconds).

I'll make a PR to add this to the docs as you can't tell without checking wiki.vg.

**Reply by KyllianGamer** [reply]
Oh, damn, I don't know how I didn't see that. Thanks!
